karma.browserstack.conf.js

```diff
@@ -23,7 +23,7 @@ module.exports = config => {
     bs_firefox_windows: {
       base: "BrowserStack",
       browser: "Firefox",
-      browser_version: "latest-beta",
+      browser_version: "84",
```

Comment:
Do we have a process for updating these version numbers? Why not use latest?

Reply:
Good call..